File: Python/problem007.py
# What is the 10 001st prime number?

i = 1
prime = 1

while i<=10001:
    prime += 1
    isPrime = 1
    for j in range(2,int(prime**0.5)+1):
        if prime % j == 0 :
            isPrime = 0
            break
    if isPrime:
        i +=1
print(prime)

File: C/problem007.c
/* What is the 10 001st prime number? */
#include <stdio.h>
int main()
{
    long long nth_prime = 0;
    long long i=1, j;
    while (nth_prime <10001)
    {   
        i++;
        int isPrime = 1;
        for (j = 2; j * j <= i;j++)
        {
            if (i % j == 0)
            {
                isPrime = 0;
                break;
            }
        }
        if (isPrime)
            nth_prime++;
    }
    printf("%lld",i);
    return 0;
}